src/components/Modal/Translate.vue

```diff
@@ -210,7 +225,7 @@ export default {
 }
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
question: Any reason you removed this CSS being scoped? Just for best practices, I know this is only a text modal, but keeping it scoped will help ensure nothing funky happens in case there are similar classes

Reply:
especially if you are adding the `!important` elements in the CSS, this indicates that other pieces of css that may not be scoped are affecting this code block, then having this be scoped helps prevent similar things in the future